packages/types/src/payment/mutations.ts

```diff
@@ -31,17 +31,40 @@ export interface CreatePaymentDTO {
   provider_id: string
   data: Record<string, unknown>
 
+  payment_session_id: string
+  payment_collection_id: string
+
   cart_id?: string
   order_id?: string
   order_edit_id?: string
   customer_id?: string
 }
 
 export interface UpdatePaymentDTO {
+  id: string
+
   cart_id?: string
   order_id?: string
   order_edit_id?: string
   customer_id?: string
+
+  captured_at?: Date
```

Comment:
thought: I don't think this should be exposed in the API. Will happen automatically upon capture
```suggestion
  captured_at?: Date
```